modules/dataquery/help/dataquery.md

```diff
@@ -1,44 +1,90 @@
+<br>
+If you click on any category, its field names and field descriptions will appear in a table.<br>
+<br>
+If the number of fields in the chosen category exceeds the display limit per page, the results may be displayed on subsequent pages within the table, accessible via the page number buttons found by scrolling toward the bottom of the page. <br>
+<br>
+Any fields that are defined for the current query are listed on the right-hand side (subject to change). If there are no fields defined for the current query, this list is not shown. Time points can be specified within the field in the “Define Fields” tab or in the “Define Filters” tab.<br>
+<br>
+The options to add fields to your query are as follows:<br>
+<br>
+Option 1: Add Using Dropdown Fields can be searched for using the list of categories of data fields. After clicking the desired instrument, a list of fields are displayed. Clicking on a field row will highlight it and add it to the list of selected fields for your current query. <br>
+<br>
+Option 2: Search Within Instrument Fields can be searched for using the search bar. Once the user starts typing, all matches will be displayed in an active list. Once the desired field is displayed, the field must be clicked to be added into the query. <br>
+<br>
+Option 3: Add All If all fields from a particular category should be included in the query, use this method. By clicking the “Add All” button above the list of categories, all fields from the category will be added. <br>
+<br>
+Option 4: Load fields from a previously saved query -- click on the "Load Saved Query" tab. User Saved Queries and Shared Saved Queries will appear in a dropdown. (User Saved Queries are queries that the current user has saved, and are listed according to Query Name. Shared Saved Queries are publicly shared, and are listed according to the user who created it and the Query Name.) <br>
+<br>
+Once a query is selected, the DQT will retrieve and load both the fields and filters defined in the saved query -- displayed in the “Define Fields” and “Define Filters” tabs. To remove fields from your query, find the field from the list (using the Search Within Instrument search bar or in the instrument list of fields) and click on the highlighted field, to de-select it. This can be confirmed by checking that the field no longer appears in your list of selected fields. <br>
+<br>
+To remove *all* currently selected fields, click “Clear Query” in the list of Fields.<br>
+<br>
+To remove only fields from a specific *category*, click the “Remove All” button above the list of categories. <br>
+<br>
+When ready, continue to the “Define Filters” tab to add filters to your query, or go directly to the “View Data” tab if your query does not require filters.<br>
+<br>
```

Comment:
I have tested this format on different browsers. This format is not correct. You need to remove the <br> and keep this line a blank line.

Reply:
Hi Shen, ok thanks.  
@knoll-alex are you ok to make these fixes?  If you're unclear, don't hesitate to ask

Reply:
This has been done! The resulting file looks good to be but hopefully it renders properly in the helptext.